js/widgets/panel.js

```diff
@@ -142,16 +142,20 @@ return $.widget( "mobile.panel", {
 			wrapper = $( "<div>" );
 			this._addClass( wrapper, "ui-panel-wrapper" );
 			wrapper = this._safelyWrap( thePage, wrapper,
-				this._page().children( ".ui-header:not(.ui-header-fixed), " +
-					".ui-content:not(.ui-popup), .ui-footer:not(.ui-footer-fixed)" ) );
+				this._page().children( ".ui-toolbar-header:not(.ui-toolbar-header-fixed), " +
+					"[data-role='toolbar'],"  +
```

Comment:
need to add in $.ui.ns here for now to support chaning that until its removed in 1.6 and will be data-ui-toolbar